first-party/jib-native-image-extension-gradle/README.md

```diff
@@ -0,0 +1,42 @@
+# Jib GraalVM Native Image Extension
+
+This extension containerizes a [GraalVM native-image](https://www.graalvm.org/docs/reference-manual/native-image/) application configured with [Native Image Gradle Plugin](https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html).
+
+The extension expects the `org.graalvm.buildtools.native` to do the heavy lifting of generating a "native image" (with the `nativeCompile` task). (The "image" in "native image" refers to an executable binary, not a container image.) Then the extension simply copies the binary, say, `<project root>/build/native/nativeCompile/com.example.mymainclass`, into a container image and sets executable bits. It also auto-sets the container image entrypoint to the binary, say, `/app/com.example.mymainclass` (unless you manually configure `container.entrypoint` in the main Jib configuration).
+However, note that this extension doesn't autoconfigure the native image name. We are required to specify it through the `imageName` property as shown in the Examples section.
+
+## Examples
+
+Check out the [general instructions](../../README.md#using-jib-plugin-extensions) for applying a Jib plugin extension.
+
+```gradle
+// should be at the top of build.gradle
+buildscript {
+  dependencies {
+    classpath('com.google.cloud.tools:jib-native-image-extension-gradle:0.1.0-SNAPSHOT')
```

Comment:
We should remember to update this after releasing.

Reply:
Yes, good call.